### ISSUE TITLE
Supporting Instructors

### ISSUE THREAD

**Issue by DanielleQuinn** [ahurford/quantitative-training-resources#5]
Hi Amy,

I wasn't sure where this would fit so I'll let you decide!

I think an important component of making this sustainable is making sure that faculty / instructors / TAs have the necessary support and are able to make changes or additions to the curriculum without having a strong background using these tools themselves, where ever possible. I have a couple of idea about how we can facilitate this:

(1) Creating modular learning activities with clear learning objectives and scaffolded exercises that can meet learners where they're at. This was touched on in the meeting today - there are some great starting points in the current Carpentries material, along with instructor notes, common troubleshooting solutions, etc. all of which will help instructors feel more comfortable / willing to incorporate an activity. Especially in cases where time is limited, these activities would ideally be built upon existing labs / projects.

(2) Provide training opportunities in the Department. We have ten or more Carpentries instructors here in St. John's and many are looking for the opportunity to offer workshops. I propose we offer regular workshops specific to the department, which normalizes developing data / coding skill sets and gives instructors the opportunity to build up their own knowledge / skill. Since at least some of the modular pieces may end up coming from the Carpentries curriculum, it'll also give them some insights into the teaching philosophy / pedagogical approaches that have worked well for that community!

**Comment by ahurford** [comment]
Yes, I think the whole repo needs reorganizing! Thanks for logging your comment, I'll try to figure how to restructure and incorporate.
